ipalib/util.py

```diff
@@ -109,7 +109,10 @@ def json_serialize(obj):
 
 def verify_host_resolvable(fqdn):
     try:
-        if not resolve_ip_addresses(fqdn):
+        addresses = resolve_ip_addresses(fqdn)
+        if addresses:
+            return addresses
```

Comment:
The return value doesn't seem to be used anywhere. This previously implicitly returned None or raised an exception.